.github/workflows/test.yml

```diff
@@ -440,7 +440,7 @@ jobs:
     needs: [ build-pr ]
     env:
       JOB_NAME: "${{ matrix.target }}-${{ matrix.ha }}-${{ matrix.gateway-mode }}-${{ matrix.ipfamily }}-${{ matrix.disable-snat-multiple-gws }}-${{ matrix.second-bridge }}-${{ matrix.ic }}"
-      OVN_HYBRID_OVERLAY_ENABLE: "${{ matrix.target == 'control-plane' || matrix.target == 'control-plane-helm' }}"
+      OVN_HYBRID_OVERLAY_ENABLE: ${{ (matrix.target == 'control-plane' || matrix.target == 'control-plane-helm') && matrix.ipfamily == 'ipv4' }} # Hybrid overlay is only supported for IPv4
```

Comment:
it might work OK for dualstack if the pod only tries to use ipv4 for the HO use case?

Reply:
Let's try!